### PR TITLE
Update license headers to 2022

### PR DESCRIPTION
**Pull request by cweitkamp**
- Update license headers to 2022

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: sat-plugin/src/main/java/org/openhab/tools/analysis/report/ReportMojo.java [hunk 1]
```diff
@@ -1,37 +1,14 @@
 /**
```
Comment (cweitkamp) [inline sat-plugin/src/main/java/org/openhab/tools/analysis/report/ReportMojo.java:1]: Was this header different by intention?

Comment (wborn) [reply sat-plugin/src/main/java/org/openhab/tools/analysis/report/ReportMojo.java:1]: Yes see https://github.com/openhab/static-code-analysis/blob/main/LICENSE

It's excluded from the check:

https://github.com/openhab/static-code-analysis/blob/7b5050dc554634e2cb0108aec041eec0c2eb80c6/tools/static-code-analysis/checkstyle/suppressions.xml#L8

Comment (kaikreuzer) [reply sat-plugin/src/main/java/org/openhab/tools/analysis/report/ReportMojo.java:1]: I think so, yes. It seems that [this code was copied](https://github.com/openhab/static-code-analysis/pull/427/files/8bd1abd56a7558b8d0c7122ad2dee8e650ec9158#diff-5895a1cf809a55b9aeb3485f43d31d3da4e121ad3c5001ebb0898611ca3bcdefR66-R71) and is licensed under BSD, so we should keep the copyright header in place.

Comment (cweitkamp) [reply sat-plugin/src/main/java/org/openhab/tools/analysis/report/ReportMojo.java]: Alright. I reverted it in 76b0970 and excluded it from further formatting.

**Review by wborn — APPROVED** [review]
Very good!

**Pull request merged**